app/account/views.py

```diff
@@ -268,3 +269,22 @@ def change_name():
         db.session.commit()
         flash('Your name has been updated.', 'form-success')
     return render_template('account/manage.html', form=form)
+
+
+@account.route('/manage/csv-settings', methods=['GET', 'POST'])
```

Comment:
no. there exists a login required, and that's enough, because Stephen asked for the admin to be able to test and update various csv uploads and settings, so I plan on reuising that for the admin view, hence, simply login is required.